Detections/MultipleDataSources/Solorigate-VM-Network.yaml

```diff
@@ -0,0 +1,77 @@
+id: ab4b6944-a20d-42ab-8b63-238426525801
+name: Solorigate Network data
```

Comment:
Since we will have 2 of these very similar but against different data sets we should be clear in the name.  I suggest something like - "Solorigate Domains Found in VM Insights"